views/package_version/show.ejs

```diff
@@ -18,11 +18,9 @@
         </select>
       </div>
         <% if(inViewerPane) { %>
+        <section data-package-name="<%=pkgVersion.package_name%>" data-type-id="<%=pkgVersion.package['type_id']%>" class="packageData" itemscope itemtype="http://schema.org/SoftwareApplication">
```

Comment:
you can actually put the data attributes in the top parent section, that's why I put the 
```
itemscope itemtype="http://schema.org/SoftwareApplication"
```
I should have been more clear